glustercli/cmd/volume-profile.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gluster/glusterd2/pkg/api"
+
+	"github.com/olekukonko/tablewriter"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Profile Info Flags
+	flagProfileInfoPeek            bool
+	flagProfileInfoIncremental     bool
+	flagProfileInfoIncrementalPeek bool
+	flagProfileInfoCumulative      bool
+	flagProfileInfoClear           bool
+)
+
+var volumeProfileCmd = &cobra.Command{
+	Use:   "profile",
+	Short: "Gluster volume profile",
+	Args:  cobra.ExactArgs(2),
+}
+
+func init() {
+	// Volume Profile Info
+	volumeProfileInfoCmd.Flags().BoolVar(&flagProfileInfoPeek, "peek", false, "Volume Profile Info Peek")
+	volumeProfileInfoCmd.Flags().BoolVar(&flagProfileInfoIncremental, "incremental", false, "Volume Profile Info Incremental")
+	volumeProfileInfoCmd.Flags().BoolVar(&flagProfileInfoIncrementalPeek, "incremental-peek", false, "Volume Profile Info Incremental Peek")
+	volumeProfileInfoCmd.Flags().BoolVar(&flagProfileInfoCumulative, "cumulative", false, "Volume Profile Info Cumulative")
+	volumeProfileInfoCmd.Flags().BoolVar(&flagProfileInfoClear, "clear", false, "Volume Profile Info Clear")
+
+	volumeProfileCmd.AddCommand(volumeProfileInfoCmd)
+
+	volumeCmd.AddCommand(volumeProfileCmd)
+}
+
+var volumeProfileInfoCmd = &cobra.Command{
+	Use:   "info <volname> [--peek|--incremental|--incremental-peek|--cumulative|--clear]",
+	Short: "Volume Profile Info",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		var err error
+		var volumeProfileInfo []api.BrickProfileInfo
+		volname := args[0]
+		option := "info"
+		if flagProfileInfoPeek {
```

Comment:
question:
what happens if we set multiple flags
```flagProfileInfoPeek``` ```flagProfileInfoIncremental```
I think you need to write some comments about who is having the highest precedence

Reply:
There isn't any kind of  priorities. The flag that comes first in the CLI command will be taken into consideration.